flixel/system/FlxSound.hx

```diff
@@ -629,6 +640,23 @@ class FlxSound extends FlxBasic
 		return Volume;
 	}
 	
+	
+	private inline function get_pitch():Float
+	{
+		return _pitch;
+	}
+	
+	private function set_pitch(v:Float):Float
+	{
+		#if (cpp || neko)
+		_channel.pitch = v;
+		#else
+		FlxG.log.warn("Pitch is only supported on OpenAL targets (i.e. cpp & neko)");
```

Comment:
Wondering if it wouldn't be better to only have this property exist  for cpp and neko in the first place. That's how we handle most of the input stuff in `FlxG` for example. Then again. other features like post processing use this approach (except for the warn() call), so I guess we're not really consistent about it...